tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/javainterop/RefTypeTests.java

```diff
@@ -388,7 +387,7 @@ public static HandleValue getHandle() {
         return new HandleValue(m);
     }
 
-    public static StringValue useHandle(HandleValue h) {
+    public static org.ballerinalang.jvm.values.api.BString useHandle(HandleValue h) {
```

Comment:
need the long name ?

Reply:
yes. org.ballerinalang.model.values.BString also imported here for assertions.

Reply:
got it.